src/vm/svml-compiler.ts

```diff
@@ -1001,20 +1001,16 @@ function transformForLoopsToWhileLoops(program: es.Program) {
       if (init!.type === 'VariableDeclaration') {
         const loopVarName = ((init as es.VariableDeclaration).declarations[0].id as es.Identifier)
           .name
+        const newLoopVarName = loopVarName + '-loop-control-var' // purposely long to reduce unintentional clash
+        const copyOfNewLoopVarName = 'copy-of-' + newLoopVarName
```

Comment:
to be safe you might use a string that is not allowed as a name in JavaScript, maybe

"*" + loopVarName + "*"

Reply:
Forgot to remove the comment, "-" is already an invalid character for identifiers

Reply:
Oh, didn't look close enough. But then why so long?

Reply:
I guess it's nice and clear for debugging. Just remove the comment, then.

Reply:
Probably not a bad thing to have long identifiers. It might also come in handy if manually debugging the VM, maybe.